File: Lab 3/Equipment.java
import java.util.Comparator;

public class Equipment {

    private String ProduceName;
    private double weight;

    public Equipment( String ProduceName , double weight ){
        this.ProduceName=ProduceName;
        this.weight=weight;
    }

    public class ProduceName implements Comparator<Equipment>{

        @Override
        public int compare(Equipment o1, Equipment o2) {
            return 0;
        }
    }


    public double getWeight() {
        return weight;
    }

    public void setWeight(double weight) {
        this.weight = weight;
    }

    public String getProduceName() {
        return ProduceName;
    }

    public void setProduceName(String produceName) {
        ProduceName = produceName;
    }

}

File: lab 13-14/model/equipment.py
class Equipment:

    def __init__(self, produce_name, weight,type):
        self.produceName = produce_name
        self.weight = weight
        self.type = type

    def __str__(self):
        return 'Produce_name = ' + str(self.produceName) + ', weight = ' + str(self.weight) +\
               ', type = ' + str(self.type)